src/core/db/models.py

```diff
@@ -61,7 +61,7 @@ class Task(Base):
     location: Mapped[str]
     link: Mapped[str]
     description = Mapped[str]
```

Comment:
Вот из-за этого ляпа у нас не создаются миграции... Поправишь? 
```suggestion
    description: Mapped[str]
```

Reply:
Как же я то не заметил... Поменяю и создам новую миграцию.